src/main/java/com/bookstore/core/dto/CreateBookRequestDto.java

```diff
@@ -5,24 +5,23 @@
 import jakarta.validation.constraints.Pattern;
 import jakarta.validation.constraints.Positive;
 import java.math.BigDecimal;
-import lombok.Data;
 
-@Data
-public class CreateBookRequestDto {
-    @NotBlank
-    private String title;
-    @NotBlank
-    private String author;
-    @NotNull
-    @Pattern(regexp = "^(?:ISBN(?:-1[03])?:? )?(?=[0-9X]{10}$|(?=(?:[0-9]+[- ]){3})"
-            + "[- 0-9X]{13}$|97[89][0-9]{10}$|(?=(?:[0-9]+[- ]){4})[- 0-9]{17}$)(?:97"
-            + "[89][- ]?)?[0-9]{1,5}[- ]?[0-9]+[- ]?[0-9]+[- ]?[0-9X]$",
-            message = "must conform to ISBN-10 or ISBN-13 formats, "
-            + "details: https://en.wikipedia.org/wiki/ISBN")
-    private String isbn;
-    @NotNull
-    @Positive
-    private BigDecimal price;
-    private String description;
-    private String coverImage;
+public record CreateBookRequestDto(
+        @NotBlank
+        String title,
+        @NotBlank
+        String author,
+        @NotNull
```

Comment:
```suggestion
public record CreateBookRequestDto(
    @NotBlank
      String title,
      @NotBlank
      String author,
      @NotNull
```

4 spaces need

Reply:
@kozhukhovsky It's record fields. Don't they need 8 spaces? They are in block (...), it seems it counts "Continuation indent".

Reply:
![2023-11-23 13_00_27-bookstore – CreateBookRequestDto java](https://github.com/VoitseshukV/bookstore/assets/67281752/4d3b0721-d290-42e1-be8f-3da99dbb03ab)
Yes, the checkstyle plugin does not like this either